### ISSUE TITLE
x axis units incorrect for Anki 2.1

### ISSUE THREAD

**Issue by matthayes** [matthayes/anki_progress_stats#16]
This is related to #15.  Anki added an arg `xunit` to `_graph` to control the units.  The old method no longer works correctly.  When selecting the past year the unit incorrectly says "d" instead of "w".  Need to deal with the Anki 2.0 vs 2.1 differences.

**Comment by matthayes** [comment]
Fixed in 0800d1cf53ec4a490a181779bcb40d551c08393e.  Released in v0.13.

**Comment by nickcmorris** [comment]
Just downloaded the latest version from https://ankiweb.net/shared/info/266436365 for use on version 2.1.20 of Anki Desktop. Still experiencing this issue where x-axis labels show "d" instead of "w" when the graph is plotted by week.

**Comment by matthayes** [comment]
Thanks I'll take another look at this.

**Comment by matthayes** [comment]
I was able to reproduce the problem with Anki 2.1.23.  I figured out the issue and uploaded a new version of the plugin (v0.14).  The problem was that Anki started using type annotations, and the way I was checking `_graph` for the presence of `xunit` no longer worked.  This code exists for compatibility with older versions of Anki.  So an error was being thrown and silently ignored.  I switched to using another method that works with type annotations.

Thanks again for raising this issue!
